packages/amplify-util-uibuilder/package.json

```diff
@@ -16,8 +16,8 @@
   "dependencies": {
     "@aws-amplify/amplify-cli-core": "4.0.5",
     "@aws-amplify/amplify-prompts": "2.7.0",
-    "@aws-amplify/codegen-ui": "2.13.1",
-    "@aws-amplify/codegen-ui-react": "2.13.1",
+    "@aws-amplify/codegen-ui": "2.13.2-ad9ba60.0",
+    "@aws-amplify/codegen-ui-react": "2.13.2-ad9ba60.0",
```

Comment:
Why there's a hash in version number ?

Reply:
its a bug fix release version

Reply:
Yeah we can do that. Going to close this pr. We need to make revisions